src/main/java/io/stargate/sgv2/jsonapi/service/schema/tables/ApiVectorType.java

```diff
@@ -18,6 +20,7 @@ public class ApiVectorType extends CollectionApiDataType {
       FROM_COLUMN_DESC_FACTORY = new ColumnDescFactory();
   public static final TypeFactoryFromCql<ApiVectorType, VectorType> FROM_CQL_FACTORY =
       new CqlTypeFactory();
+  public static final int DEFAULT_MAX_VECTOR_LENGTH = 4096;
```

Comment:
can we check this with cassandra pls

Reply:
I used 4096 because that's the maximum value we support in collections. Cassandra [supports a maximum dimension of 8K (2^13) items](https://cassandra.apache.org/doc/latest/cassandra/reference/vector-data-type.html). Maybe we could remove it and let the DB handle this maximum check?

Reply:
We should define this in a constant class, shared between Collections and Tables. Seems reasonably to follow, for now, 4096 limit (which was Cassandra limit when Collections were implemented)